cmd/yace/scraper.go

```diff
@@ -78,8 +78,6 @@ func (s *scraper) scrape(ctx context.Context, logger logging.Logger, cache sessi
 		cache,
 		exporter.MetricsPerQuery(metricsPerQuery),
 		exporter.LabelsSnakeCase(labelsSnakeCase),
-		exporter.CloudWatchAPIConcurrency(cloudwatchConcurrency),
-		exporter.TaggingAPIConcurrency(tagConcurrency),
```

Comment:
I _think_ these are still needed here?

Reply:
Oops, I forgot to put them back after a temporary refactor to hide the concurrency fully behind the client cache. I still think it might be a good thing to move but don't want to do it here.